src/components/Credentials/CredentialImage.js

```diff
@@ -33,8 +33,8 @@ const CredentialImage = ({ credential, className, onClick, showRibbon = true })
 						<img src={svgImage} alt={"Credential"} className={className} onClick={onClick} />
 					)}
 				</>
-			) : parsedCredential && parsedCredential.credentialImage.credentialImageURL && (
-				<img src={parsedCredential.credentialImage.credentialImageURL} alt={"Credential"} className={className} onClick={onClick} />
+			) : (
+				<img src={parsedCredential?.credentialImage?.credentialImageURL || 'https://picsum.photos/300/200' } alt={"Credential"} className={className} onClick={onClick} />
```

Comment:
The application now errors if the credential does not contain a credentialImageSvgTemplateURL or a credentialImageURL, or has no credentialImage at all. Also, the credentialImage can reside under `credential.vc.credentialImage` if the VC has a different format. More flexibility in retrieving the credentialImage as well as a fallback image would be helpful.